docs/topics/feed-exports.rst

```diff
@@ -429,3 +431,41 @@ format in :setting:`FEED_EXPORTERS`. E.g., to disable the built-in CSV exporter
 .. _Amazon S3: https://aws.amazon.com/s3/
 .. _botocore: https://github.com/boto/botocore
 .. _Canned ACL: https://docs.aws.amazon.com/AmazonS3/latest/dev/acl-overview.html#canned-acl
+
+.. setting:: FEED_STORAGE_BATCH_ITEM_COUNT
+
+FEED_STORAGE_BATCH_ITEM_COUNT
+-----------------------------
+Default: ``0``
+
+If assigned an integer number higher than ``0``, Scrapy generates multiple output files
+storing up to the specified number of items in each output file.
+
+When generating multiple output files, you must use at least one of the following
+placeholders in :setting:`FEED_URI` to indicate how the different output file names are
+generated:
+
+* ``%(batch_time)s`` - gets replaced by a timestamp when the feed is being created
+  (e.g. ``2020-03-28T14-45-08.237134``)
+
+* ``%(batch_id)s`` - gets replaced by the batch  sequence number of batch
+  (e.g. ``2`` for the second file)
```

Comment:
Should some kind of padding be added here? A simple numeric increment would produce unexpected results I think:
```
>>> sorted(["1", "2", "10"])
['1', '10', '2']
```

`zfill` seems like a reasonable solution, but I'm not sure about the amount of characters.

Reply:
This is a good idea, but for use zfill we should know the maximum amount of characters in %(batch_id)s in advance. If I clearly understand what you propose.

Reply:
I think the current approach is OK. Smart file browsers should be able to sort the generated files properly.

In the future, maybe we can extend the `%(batch_id)s` syntax to allow users to specify a number of leading zeroes.

Reply:
I have traveled into the future and I've seen a lot of issues about this 😛 

The problem is that most file browsers are not that smart. The only ones that worked for me were Finder on macOS and Caja on Linux Mint, and I think that wasn't the case not so long ago.
In particular, `ls` and `aws s3 ls` do not work, and I think that's a problem.

What about going with a fixed number, like 4? It's pushing the problem into the future, but given that the current code allows only one batch, I believe allowing 10000 should be more than enough for most cases, only adding 3 bytes to the existing file names.

Reply:
I wonder if we can solve it by

1) documenting that batch_id is always an integer
2) using a different format specifier, e.g.  "%(batch_id)04d" or "%(batch_id)05d"
3) explainng the issue in the docs and showing how to increase leading zero count by using a different format specifier

Reply:
That's great, I knew about stuff like `"{batch_id:04d}".format(batch_id=1)` but I wasn't aware of the percentage formatting equivalent. Seems like no code changes are necessary then, documentation is the way to go. Thanks!